net: gptp: Replace calls to power function with pre-computed values

laperie opened the pull request:
gPTP subsystem was calling pow(x,y) function with X and Y being
constants; these are replaced with the pre-computed values.

Signed-off-by: laperie <andrei.laperie@intel.com>

pfalcon reviewed (commented):
It's a wonder to me that we use floating-point here at all.

jukkar reviewed (approved):
LGTM, just fix the signed-off-by in commit msg and re-submit.

Commit:
net: gptp: Replace calls to power function with pre-computed values

gPTP subsystem was calling pow(x,y) function with X and Y being
constants; these are replaced with the pre-computed values.

Signed-off-by: Andrei Laperie <andrei.laperie@intel.com>
